site/docs/src/json/identity-providers/facebook-request.json

```diff
@@ -13,6 +13,14 @@
     "enabled" : true,
     "fields" : "email",
     "permissions" : "email,profile_image",
+    "tenantConfiguration": {
+      "e872a880-b14f-6d62-c312-cb40f22af465": {
+        "limitUserLinkCount": {
+          "enabled": true,
+          "maximumLinks": 1
```

Comment:
This is an optional configuration, so perhaps we should use the default configuration? I could go either way I guess, but if we document the example of `1` will this confuse anyone?

Reply:
Yeah...you make a good point. Perhaps better to pull this out of the json requests since its fully optional?